azure_functions_worker/dispatcher.py

```diff
@@ -640,8 +642,10 @@ async def _handle__invocation_request(self, request):
                 http_request = await http_coordinator.get_http_request_async(
                     invocation_id)
 
-                await sync_http_request(http_request, invoc_request)
-                args[fi.trigger_metadata.get('param_name')] = http_request
+                trigger_arg_name = fi.trigger_metadata.get('param_name')
```

Comment:
Can we add some unit tests for this?

Reply:
will do

Reply:
this is tested in return_route_param test, updated the unit test to test capabilities